packages/react-components/react-combobox/library/src/components/Combobox/useComboboxStyles.styles.ts

```diff
@@ -119,16 +119,15 @@ const useStyles = makeStyles({
     border: `${tokens.strokeWidthThin} solid ${tokens.colorNeutralStroke1}`,
     borderBottomColor: tokens.colorNeutralStrokeAccessible,
   },
-
   outlineInteractive: {
     '&:hover': {
       ...shorthands.borderColor(tokens.colorNeutralStroke1Hover),
-      borderBottomColor: tokens.colorNeutralStrokeAccessible,
+      borderBottomColor: tokens.colorNeutralStrokeAccessibleHover,
     },
 
-    '&:active': {
+    '&:active, &:focus-within': {
```

Comment:
Plz split this to two separate objects. The same for other files

Reply:
@layershifter good, thx!